subprojects/farmhash.wrap

```diff
@@ -0,0 +1,9 @@
+[wrap-file]
+directory = farmhash-1.1
+source_url = https://github.com/raghaven447/farmhash/archive/refs/tags/v1.1.tar.gz
```

Comment:
This is a personal fork of an archived repo that was last updated 6 years ago.  Is there a commonly-used downstream fork?